packages/xo-web/src/xo-app/home/pool-item.js

```diff
@@ -263,7 +264,8 @@ export default class PoolItem extends Component {
             <Col mediumSize={4} className={styles.itemExpanded}>
               <span>
                 {_('homePoolMaster')}{' '}
-                <Link to={`/hosts/${pool.master}`}>{poolHosts && poolHosts[pool.master].name_label}</Link>
+                <Link to={`/hosts/${pool.master}`}>{poolHosts && poolHosts[pool.master].name_label}</Link>{' '}
+                {master.productBrand} {master.version}
```

Comment:
Testing it, it looks a bit weird, with all the information stuck together. Could we have columns like this instead so that it's more readable and more consistent between hosts and pools?

Pool:
Resources - Product brand - Tags - Master

Host:
Resources - Product brand - Tags - Graphs